mlir/include/mlir/Dialect/Vector/Transforms/VectorDistribution.h

```diff
@@ -75,10 +75,18 @@ using WarpShuffleFromIdxFn =
 /// Collect patterns to propagate warp distribution. `distributionMapFn` is used
 /// to decide how a value should be distributed when this cannot be inferred
 /// from its uses.
+///
```

Comment:
Could you copy and tailor the explanation to `populateDistributeTransferWriteOpPatterns` too? That way people understand why by default we have a non-1 benefit there.